.github/actions/combine-multi-arch-images/action.yaml

```diff
@@ -54,55 +54,98 @@ runs:
     - name: Get base tags
       id: get-base-tags
       run: |
```

Comment:
Can you create a shell script file that performs this process and rewrite this action to just call it? This will improve readability and maintainability.